docs/concepts/project-invite-membership.mdx

```diff
@@ -1,74 +0,0 @@
----
-id: project-invite-membership
```

Comment:
Can you link the internal documentation link where this moved to?

Reply:
https://github.com/ory-corp/general/pull/646